packages/playwright-core/src/server/chromium/videoRecorder.ts

```diff
@@ -135,6 +135,7 @@ export class VideoRecorder {
     if (this._lastFrameBuffer) {
       const durationSec = timestamp - this._lastFrameTimestamp;
       const repeatCount = Math.max(1, Math.round(fps * durationSec));
+      this._progress.log(`pushing frame with duration ${durationSec} => repeating ${repeatCount} times`);
```

Comment:
logging frame proved to be too verbose and just cluttering the output, let's remove this?

Reply:
Oh, sure!